<ui version="4.0" >
 <class>KGpiMainWidget</class>
 <widget class="QWidget" name="KGpiMainWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>648</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="3" row="0" column="0" >
    <widget class="QGraphicsView" name="graphicsView" >
     <property name="cacheMode" >
      <set>QGraphicsView::CacheBackground</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="showRichText" >
        <property name="text" >
         <string>Show rich text</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Message timeout:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="mesTimeout" >
        <property name="maximum" >
         <number>27000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QSlider" name="opacity" >
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="3" column="0" colspan="3" >
       <widget class="QPushButton" name="changeIcon" >
        <property name="text" >
         <string>Change Iconâ€¦</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="13" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Text color:</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="13" column="1" colspan="2" >
       <widget class="KColorButton" name="textColor" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="15" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Background color:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1" colspan="2" >
       <widget class="KColorButton" name="bkgndColor" />
      </item>
      <item rowspan="2" row="6" column="0" colspan="3" >
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Replace Mode</string>
        </property>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QRadioButton" name="leavePrevious" >
           <property name="text" >
            <string>LeavePrevious</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="replacePrevious" >
           <property name="text" >
            <string>ReplacePrevious</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item rowspan="5" row="8" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Corners:</string>
        </property>
       </widget>
      </item>
      <item rowspan="5" row="8" column="1" colspan="2" >
       <widget class="QComboBox" name="cornersType" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <item>
         <property name="text" >
          <string>Square</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Sharp</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Soft</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Softest</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QPushButton" name="popupTL" >
        <property name="text" >
         <string>Popup Top-Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="popupTR" >
        <property name="text" >
         <string>Popup Top-Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="popupBL" >
        <property name="text" >
         <string>Popup Bottom-Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="popupBR" >
        <property name="text" >
         <string>Popup Bottom-Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="popupCenter" >
        <property name="text" >
         <string>Popup Center</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forceInstantHide" >
        <property name="text" >
         <string>Force Instant Hide</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forceAnimatedHide" >
        <property name="text" >
         <string>Force Animated Hide</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
